wiki/Multiplayer/en.md

```diff
@@ -3,10 +3,10 @@
 [Chat_Console#Extended_Chat_Console wikilink]: /wiki/Chat_Console/en.md#extended-chat-console "Extended chat console"
 [Beatmaps2 wikilink]: /wiki/Beatmaps/en.md#how-to-get-beatmaps-on-osu- "How to get beatmaps from osu!"
 [osu!supporter wikilink]: /wiki/osu!supporter/en.md "osu!supporter"
-[Standard wikilink]: /wiki/Game_Modes/osu!/en.md "osu! game mode"
-[Taiko wikilink]: /wiki/Game_Modes/osu!taiko/en.md "osu!taiko game mode"
-[Catch the Beat wikilink]: /wiki/Game_Modes/osu!catch/en.md "osu!catch game mode"
-[osu!mania wikilink]: /wiki/Game_Modes/osu!mania/en.md "osu!mania game mode"
+[Standard wikilink]: /wiki/Game_Modes/osu!en.md "osu! game mode"
+[Taiko wikilink]: /wiki/Game_Modes/osu!taikoen.md "osu!taiko game mode"
+[Catch the Beat wikilink]: /wiki/Game_Modes/osu!catchen.md "osu!catch game mode"
+[osu!mania wikilink]: /wiki/Game_Modes/osu!maniaen.md "osu!mania game mode"
```

Comment:
This regex needs some serious fine tune...

Not doing a review because of these errors. It would be raping octonyan to tuesday.